kubectl-cloudflow/verify/blueprint_spec_test.go

```diff
@@ -0,0 +1,625 @@
+package verify
+
+import (
+	"github.com/lightbend/cloudflow/kubectl-cloudflow/domain"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"os"
+	"reflect"
+)
+
+
+var FOO = `{
+   "type":"record",
+   "name":"FOO",
+   "fields":[
+      {
+         "name":"first",
+         "type":"string"
+      }
+   ]
+}`
+
+var BAR = `{
+   "type":"record",
+   "name":"BAR",
+   "fields":[
+      {
+         "name":"first",
+         "type":"string"
+      }
+   ]
+}`
+
+var _ = Describe("A blueprint", func() {
+	Context("if streamlets, connections and streamlet descriptors are empty", func() {
+		It("should fail verification", func() {
+			var blueprint = Blueprint{}.verify()
+			emptyProblems := []BlueprintProblem{EmptyStreamlets{}, EmptyStreamletDescriptors{}, EmptyImages{}}
+			problems := blueprint.UpdateGlobalProblems()
+			for _, p := range problems {
+				println(reflect.TypeOf(p))
```

Comment:
Does this actually print anything? In the standard test module, they redirect stdout

Reply:
It is a relic I will remove.